suites/load-testing/README.md

```diff
@@ -0,0 +1,67 @@
+# k6 Load Testing
+
+QA team is moving towards making efforts to execute load testing suite for the microservices such as Fence and IndexD (for start). The team is using a stack of k6 + Grafana + Influxdb to execute and report the load test script execution. 
```

Comment:
let's put a README.md in the `loadTests/` folder that just links to the top level `doc/loadTests.md` document, and lets build up a library of documentation in the doc/ folder

Reply:
👍